md/release-notes.md

```diff
@@ -16,6 +16,12 @@ modifying the Bonita installation.
 
 ## Feature removals
 
+## API behavior change
+
+* [addProcessComment()](https://documentation.bonitasoft.com/javadoc/api/7.7/org/bonitasoft/engine/api/ProcessRuntimeAPI.html#addProcessComment-long-java.lang.String-) method in ProcessRuntimeAPI has had a behavior change that went unnoticed some releases ago:
```

Comment:
"some releases ago" : "in Bonita 7.3", so we better understand why you mention 7.3 later on in the "what in means in practical" paragraph.